tests/assets/analytics.test.js

```diff
@@ -37,16 +37,21 @@ describe(__filename, () => {
         eventAction: action,
         eventLabel: label,
       });
-      expect(Array.from(window.dataLayer.push.mock.calls[0][0])).toEqual([
-        'event',
-        category,
-        {
+      // Ugly check but the content of the object is not the only thing that
+      // matters: pushing an array doesn't work.
+      expect(window.dataLayer.push.mock.calls[0][0].toString()).toBe(
+        '[object Arguments]'
+      );
```

Comment:
```suggestion
      // We must pass an array-like object to `dataLayer.push()` but we cannot verify
      // that we pass an object since an Array is also an Object... We can make sure that
      // we are not passing an Array, though.
      // See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments
      expect(window.dataLayer.push.mock.calls[0][0]).not.toBeInstanceOf(Array);
```

Reply:
Thanks, I was too tired yesterday evening to think about this!